bundles/org.openhab.voice.voicerss/README.md

```diff
@@ -8,6 +8,10 @@ The free version allows you to make up to 350 requests per day; for more you may
 
 For more information, see <http://www.voicerss.org/>.
 
+## Obtaining Credentials
+
+Before you can integrate this service, you need to register to http://www.voicerss.org with at least a free account to get an API key.
```

Comment:
Consider updating this URL to https://www.voicerss.org/. That will potentially avoid a redirection to https after hitting http in case [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) is not configured for the site or cached by the browser yet.

Reply:
Yes, I am going to change this URL.
Strangely, I thought I tested it without success !
This will require changes in other places too.

Reply:
Done